main.nf

```diff
@@ -0,0 +1,32 @@
+// Ensure DSL2
+nextflow.enable.dsl = 2
+
+//test params
+params.config = "/agora-data-tools/test_config.yaml"
+
+//runs test config for Agora
+process AGORA_DATA_RUN {
+
+    debug true
+
+    container "sagebionetworks/agora-data-tools"
```

Comment:
I noticed the comments about updating this container image. Once this is published to Docker Hub, you'll want to pin the tag here to avoid using the previous version.

Reply:
There currently is no versioning strategy for agora-data-tools so for now, we are pinning to `latest` (which is not ideal, but will create a JIRA ticket to implement versioning strategy.